### DISCUSSION TITLE
Aprender todos os dias as novas tecnologias ativas.

### DISCUSSION THREAD

**Discussion by feccruz** [community/community#42072] [GitHub Education]
### Select Topic Area

General

### Body

Somos seres de hábitos e por isto precisamos esta na ativa todos os dias para transformar a nossa realidade pela convivência e experiência
.

**Comment by feccruz** [comment]
Aprender todos os dias torna-se o imperativo para conquistar os nossos objetivos específicos, secundários, gerais.

**Comment by samus-aran** [comment]
Hi @feccruz , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
